### PR TITLE
Add latest updates to figure title

### PR DESCRIPTION
**Pull request by priyesh777**
## Addresses:
- Issue: https://github.com/idmc-labs/helix2.0-meta/issues/81
- Issue: https://github.com/idmc-labs/helix2.0-meta/issues/148

## Changes:
- Update figure title generation scheme

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments

## This PR contains valid:

- [ ] permission checks
- [ ] translations

### CODE REVIEW

File: src/views/Entry/EntryForm/FigureInput/index.tsx [hunk 1]
```diff
@@ -136,19 +137,25 @@ const HOUSEHOLD_SIZE = gql`
...
     const figureTypeField = figureType || '(Figure Type)';
     const figureRoleField = role || '(Figure Role)';
-    const countryField = countryInfo || '(Country)';
+    const figureCauseType = causeType || '(Cause Type)';
+    const causeField = causeInfo || '(Cause Detail)';
     const startDateField = startDateInfo || '(Start Date)';
 
-    return `${countryField} - ${figureTypeField} - ${figureRoleField} - ${causeField} on ${startDateField}`;
+    return `${capitalizeFirstLetter(locationField)},  ${capitalizeFirstLetter(countryField)} - ${totalFigureField}  ${figureTypeField} - ${figureRoleField} - ${capitalizeFirstLetter(figureCauseType)},  ${capitalizeFirstLetter(causeField)} on ${startDateField}`;
 }
```
Comment (tnagorra) [inline src/views/Entry/EntryForm/FigureInput/index.tsx:159]: Let's just not lowercase the fields to begin with.

File: src/views/Entry/EntryForm/FigureInput/index.tsx [hunk 2]
```diff
@@ -676,9 +683,39 @@ function FigureInput(props: FigureInputProps) {
...
+                    figureCauseDetail = otherSubTypeOptions?.results
+                        ?.find((item) => item.id === value.otherSubType)
+                        ?.name
+                        ?.toLowerCase();
+                }
+            }
+
+            const reportedHouseHoldFigure = Math.round(
+                (value.householdSize ?? 0) * (value.reported ?? 0),
+            );
```
Comment (tnagorra) [inline src/views/Entry/EntryForm/FigureInput/index.tsx:716]: We should create a simple utility to calculate this value.

File: src/utils/common.ts [hunk 3]
```diff
@@ -71,6 +71,14 @@ export function formatDateYmd(dateValue: string | undefined) {
     return convertedDate;
 }
 
+export function capitalizeFirstLetter(str: string) {
+    return str.charAt(0).toUpperCase() + str.slice(1);
+}
```
Comment (tnagorra) [inline src/utils/common.ts:76]: We do not need to use this function.

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: src/utils/common.ts (hunk 3 above)
Comment (priyesh777) [reply src/utils/common.ts:76]: This has been used in figure title generation

**Pull request merged**